contracts/src/v0.8/l2ep/dev/GasLimitValidator.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {Validator} from "./Validator.sol";
+
+abstract contract GasLimitValidator is Validator {
+  uint32 internal s_gasLimit;
+
+  /// @notice emitted when gas cost to spend on L2 is updated
+  /// @param gasLimit updated gas cost
+  event GasLimitUpdated(uint32 gasLimit);
+
+  /// @param l1CrossDomainMessengerAddress address the L1CrossDomainMessenger contract address
+  /// @param l2UptimeFeedAddr the address of the ScrollSequencerUptimeFeed contract address
```

Comment:
`ScrollSequencerUptimeFeed` is this scroll specific?

Reply:
Small typo - nice catch